src/functions.js

```diff
@@ -626,19 +626,117 @@ defineFunction([
 defineFunction([
     "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
     "\\check", "\\hat", "\\vec", "\\dot",
-    // We don't support expanding accents yet
-    // "\\widetilde", "\\widehat"
+    "\\widehat", "\\widetilde", "\\overrightarrow", "\\overleftarrow",
+    "\\Overrightarrow", "\\overleftrightarrow", "\\overgroup",
+    "\\overlinesegment", "\\overleftharpoon", "\\overrightharpoon",
 ], {
     numArgs: 1,
 }, function(context, args) {
     const base = args[0];
+
+    const isStretchy = !utils.contains([
+        "\\acute", "\\grave", "\\ddot", "\\tilde", "\\bar", "\\breve",
+        "\\check", "\\hat", "\\vec", "\\dot",
+    ], context.funcName);
+
+    const isShifty = !isStretchy || utils.contains([
+        "\\widehat", "\\widetilde", "\\overrightarrow", "\\overrightharpoon",
+    ], context.funcName);
```

Comment:
[Here's the discussion](https://github.com/Khan/KaTeX/issues/587#issuecomment-286767664) we had about this topic before.  I agree that your arrow looks "nicest", in that I'd want to use it as a long form of `\vec`.  As linked in the discussion, lots of people want this.  I guess what would be nice if there was a way to reproduce your arrow (approximately) with LaTeX.  Is there some `\usepackage` that would do it?  `\usepackage[a]{esvect}` is probably moderately close, though I think the aspect ratio of the arrow is different.  It's weird that `\overrightarrow` doesn't use this existing glyph used by `\vec`...  I guess we could define our own `katex.sty` package that implemented it...